thirdPage.html

```diff
@@ -0,0 +1,488 @@
+                                                                </div>
+                                                            </div>
+                                                        </div>
+                                                    </div>
+
+
+                                                            </div>
+                                                                
+                                                        </div>
+
+
+
+
+
+                                                    </div>
+                                                </div>
+                                
+                                </section>
+
+                                    <section id="contact-section">
+
+
+
+
+
+                                        <div class="container-fluid">
+                                            <div class="row">
+
+                                              <div class="col-md-1"></div>
+                                              <div class="col-md-10">
+
+                                                  <div class="container-fluid">
+                                                      <div class="row">
+
+                                                          <div class="col-md-3"><!--Solution Co.-->
+                                                          <h2>Solution Co.</h2>
+                                                          <p class="justify-text"> Donec quis bibendum dui. Vestibulum ornare mollis purus vel congue.</p>
+                                                          <p class="contact-green-button-one"><i class="fa fa-facebook btn-padding-new" aria-hidden="true"></i><i class="fa fa-twitter btn-padding-new" aria-hidden="true"></i><i class="fa fa-camera-retro" aria-hidden="true"></i>
+                                                          </div>
+                                                           <div class="col-md-3"><!--Visit office-->
+                                                          <h2 class="contact-padding">Visit office</h2>
+                                                          <p class="text-size-contact"><i class="fa fa-map-marker contact-green-button" aria-hidden="true"></i>123 Walking Street New York</p>
+                                                          <p class="text-size-contact"><i class="fa fa-phone contact-green-button" aria-hidden="true"></i>010 010 0220</p>
+                                                          <p class="text-size-contact"><i class="fa fa-envelope contact-green-button" aria-hidden="true"></i>hello@company.com</p>
+                                                          </div>
+                                                           <div class="col-md-6"><!--Contact form-->
+                                                             <form>
+                                                                    <div class="form-group">
```

Comment:
Stilizuj sad formu kao na slici. da ima u jednom redu 2 input fielda jednake duzine, a ispod jedan dugacki. To se najbolje i najpravilnije radi pomocu col-ova.